ws-nextjs-app/pages/[service]/live/[id]/Header/MaskedImage/styles.tsx

```diff
@@ -119,15 +119,16 @@ export default {
       maskSize: '100% 100%',
       maskImage: `linear-gradient(
       180deg, ${mobileImageMask})`,
+      aspectRatio: '16 /9',
```

Comment:
Is this right, I thought we were making this change to allow the header _not_ to have a fixed aspect ratio?

Reply:
Ahah, I was debating commenting on this. So this is for a UX bug on smaller screen sizes. The aspectRatio here is being applied to a div and not the image. The image still has a varying aspect ratio which is what we require.

I'm not sure the best way to explain why this works. I could add a declaration 'aspectRatio: 'auto'' to Size 4 & above to reduce confusion, but this doesn't appear to actually do anything.

Reply:
I think the name of the style might be where the confusion came for me maybe if the style was called `maskedImageWrapper` it would be clearer the styles apply to the `div` rather than the image itself?

Reply:
That is a very good point, thank you fresh eyes!